certbot/tests/main_test.py

```diff
@@ -1012,12 +1014,12 @@ def test_certonly_csr_dry_run(self):
     @mock.patch('certbot.main.client.acme_client')
     def test_revoke_with_key(self, mock_acme_client):
```

Comment:
Yeah that makes sense. Should have thought about that before already. Sorry that it took me so long for the commit but I was having a stressful week at work.